docs/guides/deploying.md

```diff
@@ -26,11 +26,34 @@ main()
   });
 ```
 
-You can run it with
+You can deploy in the `localhost` network following these steps:
+
+1.  Add in your `hardhat.config.js`
+
+    ```
+    networks: {
+      localhost: {
+        url: 'http://localhost:8545',
+      },
+      hardhat: {
+        // See its defaults
+      }
+    }
+    ```
+
+2. Start a [local node](../getting-started/#connecting-a-wallet-or-dapp-to-hardhat-network)
+
+    `npx hardhat node`
+
+3. Deploy the smart contract in the `localhost` network
```

Comment:
Maybe it's worth adding a "Open a new terminal and...", just to make it more clear.